crates/uv-settings/src/settings.rs

```diff
@@ -56,12 +56,80 @@ pub struct Options {
 #[serde(rename_all = "kebab-case")]
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 pub struct GlobalOptions {
+    /// Whether to load TLS certificates from the platform's native certificate store.
+    ///
+    /// By default, `uv` loads certificates from the bundled `webpki-roots` crate. The
```

Comment:
I thought we were explicitly not using backticks for uv anywhere.

Reply:
Haha. This is copied from the CLI... I'll have to change it there too.